WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -280,7 +280,7 @@ private Fragment getDetailFragmentForNote(Note note, int idForFragmentContainer)
             fragment = CommentDetailFragment.newInstance(note.getId(), getIntent().getStringExtra(NotificationsListFragment.NOTE_PREFILLED_REPLY_EXTRA), idForFragmentContainer);
 
             // fragment is never null at this point, and always of CommentDetailFragment type. Just add this check for safety :)
-            if ( fragment != null && fragment instanceof  CommentDetailFragment) {
+            if (fragment instanceof  CommentDetailFragment) {
```

Comment:
If we remove the `null` check here, it seems like we may as well get rid of the `instanceof` check as well (and clear away the above comment) - at the moment this line generates a redundancy warning (also there's an extra whitespace here if we do end up keeping the `instanceof`).